code/defines/procs/AStar.dm

```diff
@@ -118,30 +118,30 @@ length to avoid portals or something i guess?? Not that they're counted right no
 /PathNode
 	var/turf/source //turf associated with the PathNode
 	var/PathNode/prevNode //link to the parent PathNode
-	var/f		//A* Node weight (f = g + h)
-	var/g		//A* movement cost variable
-	var/h		//A* heuristic variable
-	var/nt		//count the number of Nodes traversed
+	var/total_node_cost		//A* Node weight (f = g + h)
```

Comment:
if you're gonna change the var names, change them in the comments as well

Reply:
or even better: remove the comments